packages/patternfly-4/react-core/src/components/Progress/Progress.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Progress/styles.css';
+import { css, getModifier } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import ProgressContainer, { ProgressMeasureLocation, ProgressStatus } from './ProgressContainer';
+
+export { ProgressMeasureLocation, ProgressStatus } from './ProgressContainer';
+
+export const ProgressVariant = {
+  sm: 'sm',
+  md: 'md',
+  lg: 'lg'
+};
+
+const propTypes = {
+  /** Classname for progress component. */
+  className: PropTypes.string,
+  /** Size variant of progress. */
+  variant: PropTypes.oneOf(Object.keys(ProgressVariant)),
+  /** Where the measure percent will be located. */
+  measureLocation: PropTypes.oneOf(Object.keys(ProgressMeasureLocation)),
+  /** Progress status type. */
+  status: PropTypes.oneOf(Object.keys(ProgressStatus)),
+  /** Title above progress. */
+  title: PropTypes.string,
+  /** Actual value of progress. */
+  value: PropTypes.number,
+  /** DOM id for progress component. */
+  id: PropTypes.string,
+  /** Minimal value of progress. */
+  valueMin: PropTypes.number,
```

Comment:
Optional: These can probably be names min/max. It would be easy to assume they apply to the value.